packages/core/src/components/VerticalNav/VerticalNavItemLabel.jsx

```diff
@@ -53,10 +55,11 @@ export class VerticalNavItemLabel extends React.PureComponent {
       onClick: this.props.onClick ? this.handleClick : undefined
     };
 
-    if (this.LabelComponent === 'a') {
-      props = Object.assign(props, this.anchorProps());
-    } else if (this.LabelComponent === 'button') {
+    if (this.LabelComponent === 'button') {
       props = Object.assign(props, this.buttonProps());
+    } else if (this.LabelComponent !== 'div') {
```

Comment:
At first I was confused by why we were singling out `div`s, but now I see that that's the default return value for the `componentType` function. Would it improve readability to do something like this?
```js
const DEFAULT_LABEL_COMPONENT_TYPE = 'div';
...
return DEFAULT_LABEL_COMPONENT_TYPE;
...
else if (this.LabelComponent !== DEFAULT_LABEL_COMPONENT_TYPE)
```
or `FALLBACK_LABEL_COMPONENT_TYPE` or some other name that makes sense?